main.go

```diff
@@ -57,6 +57,11 @@ func main() {
 			Usage:  "Artifact file",
 			EnvVar: "PLUGIN_ARTIFACT_FILE",
 		},
+		cli.StringFlag{
+			Name:   "include",
+			Usage:  "Include file patterns (comma-separated)",
+			EnvVar: "PLUGIN_INCLUDE",
```

Comment:
`PLUGIN_INCLUDE_PATTERNS`

Also please indicate what patterns are accepted. Globs/regex etc? We should support globs if possible.

Reply:
Yes we are supporting ant style globs similar to jenkins plugin, changed implementation.